pkg/services/apiserver/builder/helper.go

```diff
@@ -98,6 +100,14 @@ func SetupConfig(
 		handler := filters.WithTracingHTTPLoggingAttributes(requestHandler)
 		handler = filters.WithRequester(handler)
```

Comment:
is this needed if the [context handler also has this](pkg/services/contexthandler/contexthandler.go)?

Reply:
`filters.withRequester` is a fallback in case of dev mode where we use loopback config. And, it handles the anonymous case. Ryan knows the most about why we need the latter.

contexthandler doesn't understand anonymous users.